spec/service_models/miq_ae_service_orchestration_stack_spec.rb

```diff
@@ -32,5 +32,22 @@ module MiqAeServiceOrchestrationStackSpec
         expect(service_stack.normalized_live_status).to eq(['not_exist', 'test failure'])
       end
     end
+
+    context "refresh" do
+      before :each do
+        stack.ext_management_system = FactoryGirl.create(:ext_management_system)
+        stack.save
+      end
```

Comment:
`before(:each)` is the default, this can just be `before`.

Can you one-line this as:
```ruby
before { stack.update(:ext_management_system => FactoryGirl.create(:ext_management_system)) }
```

Reply:
sure